modules/AllBooks.js

```diff
@@ -0,0 +1,50 @@
+import Book from './Book.js';
+
+export default class AllBooks {
+  constructor() {
+    this.booksSection = document.getElementById('all-books');
+    this.titleInput = document.getElementById('title-input');
+    this.authorInput = document.getElementById('author-input');
+    this.books = [];
+    if (localStorage.getItem('localBooks')) {
+      this.books = JSON.parse(localStorage.getItem('localBooks'));
+      this.displayAllBooks();
+    }
+  }
+
+  formAddBook() {
+    const title = this.titleInput.value;
+    const author = this.authorInput.value;
+    const newBook = new Book(title, author);
+    this.books.push(newBook);
+    this.displayAllBooks();
+    this.storeInLocalStorage();
+    this.titleInput.value = '';
+    this.authorInput.value = '';
+  }
+
+  storeInLocalStorage() {
+    localStorage.setItem('localBooks', JSON.stringify(this.books));
+  }
+
+  displayAllBooks() {
+    this.books.forEach((book, index) => {
+      this.bookDetails = `
+        <div class='book-details'>
+          <h3>${book.title} by ${book.author}</h3>
+          <button class='remove-btn' id='removeBtn'>Remove</button>
+        </div>
+      `;
+      this.booksSection.innerHTML += this.bookDetails;
+      this.deleteBtn = document.getElementById('removeBtn');
+      this.deleteBtn.addEventListener('click', () => this.deleteBook(index));
+    });
+  }
```

Comment:
- [ ] when I add a book and Investigate in the local storage I didn't find anything please make sure that you click on add it add book to the list and the list is stored in the local storage